python/pyspark/sql/column.py

```diff
@@ -316,6 +316,10 @@ def isin(self, *cols):
 
     isNull = _unary_op("isNull", "True if the current expression is null.")
     isNotNull = _unary_op("isNotNull", "True if the current expression is not null.")
+    isNaN = _unary_op("isNaN", "True if the current expression is NaN.")
+    isnull = _unary_op("isNull", "True if the current expression is null.")
```

Comment:
Maybe have a comment that says this is for pandas API compat? Since skimming the code it looks a little of of place with the rest of the functions. Just a thought.